charts/dremio_v2/templates/dremio-ingress.yaml

```diff
@@ -0,0 +1,41 @@
+{{- if $.Values.ingress.enabled -}}
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: dremio
+  labels:
+    app: dremio
+    {{- include "dremio.service.labels" $ | nindent 4 }}
+{{- if .Values.ingress.labels }}
+{{ toYaml $.Values.ingress.labels | indent 4 }}
+{{- end }}
+  {{- if $.Values.ingress.annotations }}
+  annotations:
+    {{- range $key, $value := $.Values.ingress.annotations }}
+    {{ $key }}: {{ tpl $value $ | quote }}
+    {{- end }}
+  {{- end }}
+{{- end }}
+spec:
+  {{- if $.Values.ingress.ingressClassName }}
+  ingressClassName: {{ $.Values.ingress.ingressClassName }}
+  {{- end -}}
+{{- if .Values.ingress.tls }}
+  tls:
+{{ tpl (toYaml $.Values.ingress.tls) $ | indent 4 }}
+  rules:
+  {{- if $.Values.ingress.hosts  }}
+  {{- range $.Values.ingress.hosts }}
+    - host: {{ .name }}
+      http:
+        paths:
+          - path: /
+            pathType: Prefix
+            backend:
+              service:
+                name: dremio-client
```

Comment:
Came here just by accident, and i dont know the dremio application, but shouldnt this be {{ .name }} ?